### DISCUSSION TITLE
希望可以支持把进度集和点评集放在首页

### DISCUSSION THREAD

**Discussion by Astral-afk** [WatsonYao/nian_app#75] [Q&A] [answered]
不是添加桌面快捷入口，是可以作为记本和其他标准集、清单集一样摆在首页上那种，因为也是比较常用的功能，而且首页上很多记本看起来超有成就感哎･ﾟ(ﾉヮ´ ) ​​​

**Comment by WatsonYao** [comment, marked as answer]
记本和集子是两套系统，桌面有小组件可以直接进入各类集子。
